src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import duke.commands.*;
+import duke.tasks.TaskType;
+
+/**
+ * Class to parse user input and process it.
+ */
+public class Parser {
+
+    static final String BYE = "bye";
+    static final String LIST = "list";
+    static final String TODO = "todo";
+    static final String DDL = "deadline";
+    static final String EVENT = "event";
+    static final String DELETE = "delete";
+    static final String DONE = "done";
+    static final String FIND = "find";
```

Comment:
I really like the extensive use of final variables here as it is easier to amend code in the future. There is also less likelihood of typing errors.